README.md

```diff
@@ -45,7 +45,7 @@ If you're interested in finding out more about Home Assistant Operating System a
 - **Security:**
   - [AppArmor](https://apparmor.net/) Linux kernel security module
 
-If you don't have experience with these, embedded systems, buildroot or the build process for Linux distributions, then please read up on these topics. The rest of the documentation in this project is for developers and assumes you have experience of embedded systems or a strong understanding of the internal workings of operating systems.
+If you don't have experience with these, embedded systems, buildroot, or the build process for Linux distributions, then please read up on these topics. The rest of the documentation in this project is for developers and assumes you have experience with embedded systems or a strong understanding of the internal workings of operating systems.
```

Comment:
Adding this comma is incorrect. We don't use the oxford comma in the documentation for our project.

See: <https://developers.home-assistant.io/docs/documenting/standards#general-documentation>

Reply:
@frenck Thanks for the review. Make sense, updated in [2c3d53c8c430e9ced0cba84dde98197090039111](https://github.com/home-assistant/operating-system/pull/930/commits/2c3d53c8c430e9ced0cba84dde98197090039111)